backend/src/main/scala/cromwell/backend/backend.scala

```diff
@@ -69,4 +71,5 @@ final case class AttemptedLookupResult(name: String, value: Try[WdlValue]) {
   def toPair = name -> value
 }
 
+// FIXME: Remove
```

Comment:
It's used only in the JABJEA tests that are currently `ignore`d... I've added a tech talk to the board for tomorrow